teuthology/run.py

```diff
@@ -226,9 +226,13 @@ def get_initial_tasks(lock, config, machine_type):
         init_tasks.extend([
             {'pcp': None},
             {'selinux': None},
-            {'ansible.cephlab': None},
-            {'clock': None}
         ])
+    if config.get('ceph_cm_ansible', True):
+        init_tasks.append({'ansible.cephlab': None})
+
+    # sync_clock: 'clock' or 'clock.check'
+    sync_clock = config.get('sync_clock', 'clock')
+    init_tasks.append({sync_clock: None})
```

Comment:
Could the config option be called `sync_clock_task` instead of `sync_clock`?

Reply:
@smithfarm easy, check the next commit out